code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -156,6 +156,37 @@
 	log_say("[src.real_name]/[src.key] : [text]")
 
 
+/mob/living/proc/guardian_recall()
+	set name = "Recall Guardian"
+	set category = "Guardian"
+	set desc = "Forcibly recall your guardian."
+	for(var/mob/M in mob_list)
```

Comment:
The mob list loop is to avoid having to set some awful snowflake var for humans to track if they have a guardian or not.